### DISCUSSION TITLE
Create sub-domain

### DISCUSSION THREAD

**Discussion by SylenthMen** [vercel/community#5372] [Help]
### Summary

Hi, I want to create the portfolio.sylenthmen.site subdomain, but I can't find a way to do it.

### Steps to Reproduce

portfolio.sylenthmen.site

**Comment by z4nr34l** [comment]
1. Add this subdomain in your vercel project;
2. Go to your DNS provider (for example Cloudflare)
3. Add Vercel DNS records which you prefer - A/CNAME/NS;
4. (CF only step) Disable CF proxy;
5. Wait a few seconds for DNS propagation and SSL generation;
6. If on Vercel there is info that domain is ready - just visit it.

**Reply by SylenthMen** [reply]
Thanks for the help
